src/components/Editor/Preview/ReactPopup.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { Component } from "react";
+
+import "./ReactPopup.css";
+
+class ReactPopup extends Component {
+  props: {
+    roots: Array<Object>
+  };
+
+  renderComponentAttributes(root: Object) {
+    if (!("preview" in root.contents.value)) {
+      return <span className="attrKey">none</span>;
+    }
+    const attributes = root.contents.value.preview.ownProperties;
+
+    return Object.keys(attributes).map(key => {
+      const val = attributes[key].value;
+      let content = "";
+
+      switch (typeof val) {
+        case "object":
+          if (val.class === "Function") {
+            content = `${val.displayName}()`;
+          } else {
+            content = val.type;
+          }
+          break;
+        case "string":
+          content = `"${val}"`;
+          break;
+        default:
+          content = attributes[key].value.toString();
+      }
+
+      return (
+        <li key={key}>
```

Comment:
do we have any errors thrown from this in the console? (asking because I can't test it at the moment 😅 )